woocommerce/woocommerce
# Update delete all to include trash

### Changes proposed in this Pull Request:

This PR updates the `deleteAllObjects` function to also delete trashed objects. It also updates the product import test to use the api to delete the imported products.

### How to test the changes in this Pull Request:

1. Run E2E tests in `trunk`
2. Note that there are products in the trash
3. Switch to this branch
4. Run `npm run build:packages`
5. Run the merchant tests `npx wc-e2e tests/e2e/specs/wp-admin/merchant.js`
6. There will be published products left from the tests but the trashed products should be deleted 

### Changelog entry

> N/A
